exercises/lists/en.md

```diff
@@ -69,6 +69,6 @@ Try to write your own list. Please create new file and create an unordered list
 - Four
 - Five
 
-If you have done, please verify your solution.
+When you are done, please verify your solution with: how-to-markdown verify [yourFileName].md
```

Comment:
Please, remove `with: how-to-markdown verify [yourFileName].md` — these lines are inserted by workshopper automatically.